syntax = "proto3";

package RComm;

message DiscoveryRequest {
  string team_name = 1; 
  uint32 robot_id = 2;   
}

message DiscoveryResponse {
  string team_name = 1; 
  bool team_color = 2;  
}

message SSLSpeed {
  uint32 id = 1;
  double vx = 2;
  double vy = 3;
  double vw  = 4;
  bool front  = 5;
  bool chip  = 6;
  bool charge  = 7;
  double kickStrength = 8;
  bool dribbler = 9;
  double dribSpeed = 10;
}